src/applications/disability-benefits/686c-674/config/form.js

```diff
@@ -76,7 +78,6 @@ const formConfig = {
   title: 'Add or remove dependents from your VA benefits',
   subTitle: 'VA Form 21-686c (and 21-674)',
   defaultDefinitions: { ...fullSchema.definitions },
-  transformForSubmit: customTransformForSubmit,
```

Comment:
Do we still need this transformForSubmit?

Reply:
nope, I removed this